File: trail1/홀수만 더하기/add-only-odd-numbers.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        int ssum = 0;
        for (int i=0; i<N; i++) {
            int a = int.Parse(Console.ReadLine());
            if (a % 3 == 0 && a % 2 == 1) {ssum += a;}
        }
        Console.WriteLine(ssum);
    }
}

File: trail1/홀수만 더하기/add-only-odd-numbers.c
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);
    int ssum = 0;
    for (int i=0; i<N; i++) {
        int a;
        scanf("%d", &a);
        if (a % 3 == 0 && a % 2 == 1) {ssum += a;}
    }
    printf("%d", ssum);
    return 0;
}